README.md

```
@@ -1,33 +1,42 @@
+```
 
-### Fearless Concurrency
+### Installation From Crates.io
 
-<!-- Message passing: *"Do not communicate by sharing memory; instead, share memory by communicating."* -->
+This feature will be available soon.
+
+## Usage
+
+An example of how to use the library is provided in the `examples` directory.
+- `example1.rs` demonstrates how to randomly create a network, train it to memorize a random spike train, and simulate the network.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)